homeassistant/setup.py

```diff
@@ -176,15 +179,15 @@ def log_error(msg: str, link: bool = True) -> None:
         for entry in hass.config_entries.async_entries(domain):
             await entry.async_setup(hass, integration=integration)
 
-    hass.config.components.add(component.DOMAIN)  # type: ignore
+    hass.config.components.add(domain)
```

Comment:
Do we use `component.DOMAIN` in other places, or does this mean we don't need to define `DOMAIN` anymore?

Reply:
Yes because of manifest

Reply:
Yeah, it's optional now. I think that it's still good to keep around and use it to as a key for `hass.data` etc, because it allows for people to easily copy an integration and change the `DOMAIN` value to make a copy.